puppet/modules/kubernetes/templates/kubelet.service.erb

```diff
@@ -70,10 +71,8 @@ ExecStart=<%= scope['kubernetes::_dest_dir'] %>/kubelet \
 <% end -%>
 <% if scope.function_versioncmp([scope['kubernetes::version'], '1.6.0']) >= 0 -%>
   --cgroup-driver=<%= @cgroup_driver %> \
-<% if not @osfamily.nil? and @osfamily == 'RedHat' -%>
-  --runtime-cgroups=/systemd/system.slice \
-  --kubelet-cgroups=/systemd/system.slice \
-<% end -%>
+  --runtime-cgroups=/podruntime.slice \
+  --kubelet-cgroups=/system.slice \
```

Comment:
There's seems to be no documentation on what the purpose of `--kubelet-cgroups` and `--runtime-cgroups` are, but from [here](https://kubernetes.io/docs/reference/generated/kubelet/) the kubelet is meant to be run in the cgroups referred to by ` --kubelet-cgroups` but the kubelet service unit assigns the kubelet to the `podruntime` slice. From what I understand, a process can only live in exactly one cgroup in a particular hierarchy, so these two settings seems to contradict eachother